code/datums/components/curse_of_hunger.dm

```diff
@@ -148,7 +148,8 @@
 	var/affecting = cursed.get_bodypart(BODY_ZONE_CHEST)
 	cursed.apply_damage(60, BRUTE, affecting)
 	hunger = 0
-	poison_food_tolerance = min(poison_food_tolerance += 1, FULL_HEALTH)
+	var/healed_food_tolerance = poison_food_tolerance += 1
```

Comment:
**ARM PLEASE STOP USING THE VALUE OF `+=` OR `++`**

Reply:
`poison_food_tolerance = min(healed_food_tolerance + 1, FULL_HEALTH)`

is ALL you need

Reply:
i suffer from stupid?